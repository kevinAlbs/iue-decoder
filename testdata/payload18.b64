Er0AAAADdHMAhQAAAANlAH0AAAAFZAAgAAAAACgO2rQZB2PR9Bg9OD+IMHc4WcOnShMWEJTm/UTnOQ/tBXMAIAAAAADDyJgO0R5j7BmRBLyaCIkyLJ64DQDu4LUUjcg/fnittwVsACAAAAAAh3MyKiuebAiIbba8ZbRv/dZGUeiklACp5V/1vFUNRb8AABJjbQArAAAAAAAAAAhjZgABCGRmAAADcHMAFQAAABB1YgAUAAAAEGxiAAIAAAAAAA==