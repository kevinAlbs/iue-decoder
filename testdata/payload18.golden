BlobSubtype=18 (FLE2IndexedTextEncryptedValue)
EDCTextExactDerivedFromDataToken=280edab4190763d1f4183d383f8830773859c3a74a13161094e6fd44e7390fed ("d":{"$binary":{"base64":"KA7atBkHY9H0GD04P4gwdzhZw6dKExYQlOb9ROc5D+0=","subType":"00"}})
ESCTextExactDerivedFromDataToken=c3c8980ed11e63ec199104bc9a0889322c9eb80d00eee0b5148dc83f7e78adb7 ("s":{"$binary":{"base64":"w8iYDtEeY+wZkQS8mgiJMiyeuA0A7uC1FI3IP354rbc=","subType":"00"}})
ServerTextExactDerivedFromDataToken=8773322a2b9e6c08886db6bc65b46ffdd64651e8a49400a9e55ff5bc550d45bf ("l":{"$binary":{"base64":"h3MyKiuebAiIbba8ZbRv/dZGUeiklACp5V/1vFUNRb8=","subType":"00"}})
MaxContentionCounter=43 ("cm":43)
CaseFolding=true ("cf":true)
DiacriticFolding=false ("df":false)
Prefix Upper bound=20 ("ub":20)
Prefix Lower bound=2 ("lb":2)
